### PR TITLE
UICHKIN-384 added request.requestDate token

### PR DESCRIPTION
**Pull request by rgupta-nla**
## Purpose
Update staff slips to have the request date

## Approach
add new token for request date

## Refs
https://issues.folio.org/browse/UICHKIN-384

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.
image

### CODE REVIEW

**Review by artem-blazhko — CHANGES_REQUESTED** [review]
Could you fix ESLint errors?
And could you add appropriate changes to CHANGELOG.md file?

**Review by zburke — CHANGES_REQUESTED** [review]
WRT the PR itself, not the code change: changes in the ui-checkin GitHub repo need tickets in the corresponding Jira project, UICHKIN. Please move CIRC-1784 into the correct project. As is, this ui-checkin feature will be affiliated with a mod-circulation release, which is incorrect.

**Comment by rgupta-nla** [comment]
> WRT the PR itself, not the code change: changes in the ui-checkin GitHub repo need tickets in the corresponding Jira project, UICHKIN. Please move CIRC-1784 into the correct project. As is, this ui-checkin feature will be affiliated with a mod-circulation release, which is incorrect.

The code change is spread across four different repositories. 
mod-circulation
ui-checkin
ui-requests
ui-circulation

Are you suggesting I create three more UIXXX tickets?

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * Populate the token "requester.departments" in the hold, request delivery and transit slips, with the data provided by the backend in the ui-checkin module. Refs UICHKIN-350.
 * Populate the token "currentDateTime" in the hold, request delivery and transit slips, with the data provided by the backend in the ui-checkin module. Refs UICHKIN-348.
 * Also support `circulation` `14.0`. Refs UICHKIN-381.
+* Added requestDate token. Refs CIRC-1784.
```
Comment (Dmitriy-Litvinenko) [inline CHANGELOG.md:11]: Could you please update CHANGELOG.md how i understood we should change from  `Added requestDate token. Refs CIRC-1784.` to `*  Added requestDate token. Refs UICHKIN-384.`

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Review by zburke — APPROVED** [review]

**Comment by zburke** [comment]
@rgupta-nla : please remember to squash when you merge in order to keep the commit history tidy. Thank you!

**Pull request merged**